map/framework.cpp

```diff
@@ -462,6 +465,17 @@ Framework::Framework()
   LOG(LINFO, ("Editor initialized"));
 
   m_trafficManager.SetCurrentDataVersion(m_storage.GetCurrentDataVersion());
+
+#ifdef OMIM_OS_ANDROID
+  ZipFileReader::UnzipFile(GetPlatform().ResourcesDir(),
```

Comment:
It's not a good thing how storage reads from sd-card countries file. So, my question is the same: are you sure that we don't need a worker thread for IO operations on startup?